LibCarla/source/carla/trafficmanager/BatchControlStage.cpp

```diff
@@ -71,8 +73,6 @@ void BatchControlStage::DataSender() {
     if (commands != nullptr) {
       episode_proxy_bcs.Lock()->ApplyBatch(*commands.get(), false);
     }
-    // Limiting updates to 100 frames per second.
-    std::this_thread::sleep_for(10ms);
```

Comment:
For threads either signal wait or sleep (even very very low amount of time.) is a good practice that gives it system a cancellation point.

Reply:
We want to keep throughput as high as possible in asynchronous mode. Is it still good to have a small sleep? If so, we can have maybe a sleep statement for a microsecond?

Reply:
Yes, nanosecond sleep also fine ... have it.